fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersionsIT.java

```diff
@@ -247,6 +247,36 @@ public void testVersionHeaders() throws IOException {
     }
 
     @Test
+    public void testVersionListHeaders() throws IOException {
+        final String id = getRandomUniqueId();
+        createObjectAndClose(id);
+        enableVersioning(id);
+
+        final Link RDF_SOURCE_LINK = fromUri(RDF_SOURCE.getURI()).rel(type.getLocalName()).build();
+
+        final HttpHead head = new HttpHead(serverAddress + id + "/fcr:versions");
+
+        try (final CloseableHttpResponse response = execute(head)) {
+            assertEquals(OK.getStatusCode(), getStatus(response));
+
+            final Collection<String> linkHeaders = getLinkHeaders(response);
+
+            final Set<Link> resultSet = linkHeaders.stream().map(Link::valueOf).flatMap(link -> {
+                final String linkRel = link.getRel();
+                final URI linkUri = link.getUri();
+                if (linkRel.equals(RDF_SOURCE_LINK.getRel()) && linkUri.equals(RDF_SOURCE_LINK.getUri())) {
+                    // Found nonRdfSource!
```

Comment:
Just noticed some copy/pasta. I'll wait for any code review comments before I push a commit to correct this.